data/issues/emissions/emission_data_calculations.py

```diff
@@ -1,21 +1,34 @@
+# pylint: disable=invalid-name
 # -*- coding: utf-8 -*-
 
 import datetime
 from dateutil.relativedelta import relativedelta
 import numpy as np
-import pandas as pd
+
 from issues.emissions.historical_data_calculations import get_n_prep_data_from_smhi
 from issues.emissions.approximated_data_calculations import calculate_approximated_historical
 from issues.emissions.trend_calculations import calculate_trend_coefficients, calculate_trend
 
+# Numbers below from
+# https://www.cemus.uu.se/wp-content/uploads/2023/12/Paris-compliant-carbon-budgets-for-Swedens-counties-.pdf
+# Found in tables on page 8 and 19
```

Comment:
There's no table on page 8 of that report. Did you mean the table on page 6 or 10?

Or wait a minute... Maybe this is dependent on which PDF reader I use... Not sure.

Also, it's a provisional report. Have they published the official version?